### DISCUSSION TITLE
The realm master disappears from the list of realms on keycloak 26.0.x

### DISCUSSION THREAD

**Discussion by sebacla** [keycloak/keycloak#35370] [Q&A] [answered]
Hey,

I have just updated Keycloak 25.06 to Keycloak 26.0.6. I have the same behavior on other KC 26.0.x
When I log on the console as a user with admin rights, I end up on the master realm. When I select another realm, I can no longer return to the master realm. It is no longer in the list of realms. I have to change the url to access it again. Is this a deliberate behavior? 

Best regards,

**Comment by jonkoops** [comment, marked as answer]
I believe this is a bug that might have been fixed under #35128, it should be backported to the next patch release.

**Reply by sebacla** [reply]
I just reduced the number of realms and the master realm magically popped up again. Thanks
